### ISSUE TITLE
[Bug]:  Data errors cause bvt failure

### ISSUE THREAD

**Issue by sukki37** [matrixorigin/matrixone#7589]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):  https://github.com/matrixorigin/matrixone/actions/runs/3892119494
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

bvt log error
```
[ERROR]
[SCRIPT   FILE]: /home/runner/work/matrixone/matrixone/test/distributed/cases/dtype/text_blob.sql
[ROW    NUMBER]: 65
[SQL STATEMENT]: select * from text_01 where t2 > (select t2 from text_05 where t1='789');
[EXPECT RESULT]:
t1    t2    t3
yef&&190     wwww     983
123    7834    commmmmmment
[ACTUAL RESULT]:
t1    t2    t3
yef&&190     wwww     983
yef&&190     wwww     983
yef&&190     wwww     983
123    7834    commmmmmment
```

Following is the related mo log; full log can download from https://github.com/matrixorigin/matrixone/actions/runs/3892119494
```
cn-0  | 2023/01/11 11:04:32.830157 +0000 INFO frontend/util.go:500 query trace {"connection_id": 1006, "query": "select * from text_01 where t2 > (select t2 from text_05 where t1='789');", "session_info": "connectionId 1006"}
cn-0  | 2023/01/11 11:04:32.830301 +0000 DEBUG cn-service util/log.go:69 txn created {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "txn": "57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0"}
cn-0  | 2023/01/11 11:04:32.830363 +0000 DEBUG cn-service util/log.go:33 txn read {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "txn": "57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0"}
cn-0  | 2023/01/11 11:04:32.830381 +0000 DEBUG cn-service util/log.go:118 txn send requests {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "requests": "[0: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F-0/<O:1000-D:457>/=><2-262146-0-dn:41010>]"}
cn-0  | 2023/01/11 11:04:32.830402 +0000 DEBUG cn-service morpc/backend.go:440 write request {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "remote": "dn:41010", "backend-id": "3172dc52-56ac-4074-afcc-175c45e444d7", "request-id": 67099, "request": "67099: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F-0/<O:1000-D:457>/=><2-262146-0-dn:41010>"}
cn-0  | 2023/01/11 11:04:32.831025 +0000 DEBUG cn-service morpc/backend.go:591 read response {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "remote": "dn:41010", "backend-id": "3172dc52-56ac-4074-afcc-175c45e444d7", "request-id": 67099, "response": "67099: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F:0/D:834"}
cn-0  | 2023/01/11 11:04:32.831079 +0000 DEBUG cn-service util/log.go:143 txn received responses {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "responses": "[67099: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F:0/D:834]"}
cn-0  | 2023/01/11 11:04:32.831221 +0000 DEBUG cn-service util/log.go:33 txn read {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "txn": "57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0"}
cn-0  | 2023/01/11 11:04:32.831285 +0000 DEBUG cn-service util/log.go:118 txn send requests {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "requests": "[0: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F-0/<O:1000-D:457>/=><2-262146-0-dn:41010>]"}
cn-0  | 2023/01/11 11:04:32.831315 +0000 DEBUG cn-service morpc/backend.go:440 write request {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "remote": "dn:41010", "backend-id": "3172dc52-56ac-4074-afcc-175c45e444d7", "request-id": 67100, "request": "67100: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F-0/<O:1000-D:457>/=><2-262146-0-dn:41010>"}
cn-0  | 2023/01/11 11:04:32.831885 +0000 DEBUG cn-service morpc/backend.go:591 read response {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "remote": "dn:41010", "backend-id": "3172dc52-56ac-4074-afcc-175c45e444d7", "request-id": 67100, "response": "67100: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F:0/D:834"}
cn-0  | 2023/01/11 11:04:32.831937 +0000 DEBUG cn-service util/log.go:143 txn received responses {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "responses": "[67100: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F:0/D:834]"}
cn-0  | 2023/01/11 11:04:32.832105 +0000 DEBUG cn-service util/log.go:33 txn read {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "txn": "57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0"}
cn-0  | 2023/01/11 11:04:32.832144 +0000 DEBUG cn-service util/log.go:118 txn send requests {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "requests": "[0: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F-0/<O:1000-D:457>/=><2-262146-0-dn:41010>]"}
cn-0  | 2023/01/11 11:04:32.832174 +0000 DEBUG cn-service morpc/backend.go:440 write request {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "remote": "dn:41010", "backend-id": "3172dc52-56ac-4074-afcc-175c45e444d7", "request-id": 67101, "request": "67101: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F-0/<O:1000-D:457>/=><2-262146-0-dn:41010>"}
cn-0  | 2023/01/11 11:04:32.832757 +0000 DEBUG cn-service morpc/backend.go:591 read response {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "remote": "dn:41010", "backend-id": "3172dc52-56ac-4074-afcc-175c45e444d7", "request-id": 67101, "response": "67101: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F:0/D:834"}
cn-0  | 2023/01/11 11:04:32.832807 +0000 DEBUG cn-service util/log.go:143 txn received responses {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "responses": "[67101: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F:0/D:834]"}
cn-0  | 2023/01/11 11:04:32.832970 +0000 DEBUG frontend/util.go:520 dbName text_blob tableNames [%!%mo_increment_columns text_01 text_03 text_04 text_05] connectionId 1006
cn-0  | 2023/01/11 11:04:32.833038 +0000 DEBUG frontend/util.go:520 dbName text_blob tableNames [%!%mo_increment_columns text_01 text_03 text_04 text_05] connectionId 1006
cn-0  | 2023/01/11 11:04:32.833132 +0000 DEBUG frontend/util.go:520 dbName text_blob tableNames [%!%mo_increment_columns text_01 text_03 text_04 text_05] connectionId 1006
cn-0  | 2023/01/11 11:04:32.833234 +0000 DEBUG frontend/util.go:520 dbName text_blob tableNames [%!%mo_increment_columns text_01 text_03 text_04 text_05] connectionId 1006
cn-0  | 2023/01/11 11:04:32.833377 +0000 DEBUG frontend/util.go:520 dbName text_blob tableNames [%!%mo_increment_columns text_01 text_03 text_04 text_05] connectionId 1006
cn-0  | 2023/01/11 11:04:32.833447 +0000 DEBUG frontend/util.go:520 dbName text_blob tableNames [%!%mo_increment_columns text_01 text_03 text_04 text_05] connectionId 1006
cn-0  | 2023/01/11 11:04:32.833598 +0000 INFO frontend/session.go:320 session uuid: 69c4f2e5-919f-11ed-ae6c-0242b3130005 -> background session uuid: ba36d243-919f-11ed-ae6c-0242b3130005
cn-0  | 2023/01/11 11:04:32.833652 +0000 DEBUG cn-service util/log.go:69 txn created {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "txn": "a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0"}
cn-0  | 2023/01/11 11:04:32.833735 +0000 DEBUG cn-service util/log.go:33 txn read {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "txn": "a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0"}
cn-0  | 2023/01/11 11:04:32.833771 +0000 DEBUG cn-service util/log.go:118 txn send requests {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "requests": "[0: <a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0>/Read/F-0/<O:1000-D:457>/=><2-262146-0-dn:41010>]"}
cn-0  | 2023/01/11 11:04:32.833803 +0000 DEBUG cn-service morpc/backend.go:440 write request {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "remote": "dn:41010", "backend-id": "3172dc52-56ac-4074-afcc-175c45e444d7", "request-id": 67102, "request": "67102: <a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0>/Read/F-0/<O:1000-D:457>/=><2-262146-0-dn:41010>"}
cn-0  | 2023/01/11 11:04:32.834398 +0000 DEBUG cn-service morpc/backend.go:591 read response {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "remote": "dn:41010", "backend-id": "3172dc52-56ac-4074-afcc-175c45e444d7", "request-id": 67102, "response": "67102: <a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0>/Read/F:0/D:834"}
cn-0  | 2023/01/11 11:04:32.834451 +0000 DEBUG cn-service util/log.go:143 txn received responses {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "responses": "[67102: <a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0>/Read/F:0/D:834]"}
cn-0  | 2023/01/11 11:04:32.834623 +0000 DEBUG cn-service util/log.go:33 txn read {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "txn": "a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0"}
cn-0  | 2023/01/11 11:04:32.834657 +0000 DEBUG cn-service util/log.go:118 txn send requests {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "requests": "[0: <a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0>/Read/F-0/<O:1000-D:457>/=><2-262146-0-dn:41010>]"}
cn-0  | 2023/01/11 11:04:32.834723 +0000 DEBUG cn-service morpc/backend.go:440 write request {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "remote": "dn:41010", "backend-id": "3172dc52-56ac-4074-afcc-175c45e444d7", "request-id": 67103, "request": "67103: <a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0>/Read/F-0/<O:1000-D:457>/=><2-262146-0-dn:41010>"}
cn-0  | 2023/01/11 11:04:32.835241 +0000 DEBUG cn-service morpc/backend.go:591 read response {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "remote": "dn:41010", "backend-id": "3172dc52-56ac-4074-afcc-175c45e444d7", "request-id": 67103, "response": "67103: <a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0>/Read/F:0/D:834"}
cn-0  | 2023/01/11 11:04:32.835291 +0000 DEBUG cn-service util/log.go:143 txn received responses {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "responses": "[67103: <a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0>/Read/F:0/D:834]"}
cn-0  | 2023/01/11 11:04:32.835473 +0000 DEBUG cn-service util/log.go:33 txn read {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "txn": "a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0"}
cn-0  | 2023/01/11 11:04:32.835510 +0000 DEBUG cn-service util/log.go:118 txn send requests {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "requests": "[0: <a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0>/Read/F-0/<O:1000-D:457>/=><2-262146-0-dn:41010>]"}
cn-0  | 2023/01/11 11:04:32.835538 +0000 DEBUG cn-service morpc/backend.go:440 write request {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "remote": "dn:41010", "backend-id": "3172dc52-56ac-4074-afcc-175c45e444d7", "request-id": 67104, "request": "67104: <a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0>/Read/F-0/<O:1000-D:457>/=><2-262146-0-dn:41010>"}
cn-0  | 2023/01/11 11:04:32.836099 +0000 DEBUG cn-service morpc/backend.go:591 read response {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "remote": "dn:41010", "backend-id": "3172dc52-56ac-4074-afcc-175c45e444d7", "request-id": 67104, "response": "67104: <a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0>/Read/F:0/D:834"}
cn-0  | 2023/01/11 11:04:32.836149 +0000 DEBUG cn-service util/log.go:143 txn received responses {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "responses": "[67104: <a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0>/Read/F:0/D:834]"}
cn-0  | 2023/01/11 11:04:32.836299 +0000 INFO frontend/util.go:500 query trace status {"connection_id": 4294967295, "statement": "start transaction", "status": "success", "span": {"trace_id": "ba36d41e-919f-11ed-ae6c-0242b3130005", "kind": "statement"}, "session_info": ""}
cn-0  | 2023/01/11 11:04:32.836359 +0000 DEBUG frontend/util.go:520 CommitTxn txnId:a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0 
cn-0  | 2023/01/11 11:04:32.836416 +0000 DEBUG cn-service util/log.go:51 txn commit {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "txn": "a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0"}
cn-0  | 2023/01/11 11:04:32.836448 +0000 DEBUG frontend/util.go:520 CommitTxn exit txnId:a9514531bb80450596d43152ad1546b2/Active/S:1673435072833649382-0 
cn-0  | 2023/01/11 11:04:32.836473 +0000 INFO frontend/util.go:500 query trace status {"connection_id": 4294967295, "statement": "commit", "status": "success", "span": {"trace_id": "ba373de8-919f-11ed-ae6c-0242b3130005", "kind": "statement"}, "session_info": ""}
cn-0  | 2023/01/11 11:04:32.836615 +0000 DEBUG cn-service util/log.go:33 txn read {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "txn": "57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0"}
cn-0  | 2023/01/11 11:04:32.836650 +0000 DEBUG cn-service util/log.go:118 txn send requests {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "requests": "[0: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F-0/<O:1000-D:463>/=><2-262146-0-dn:41010>]"}
cn-0  | 2023/01/11 11:04:32.836681 +0000 DEBUG cn-service morpc/backend.go:440 write request {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "remote": "dn:41010", "backend-id": "3172dc52-56ac-4074-afcc-175c45e444d7", "request-id": 67105, "request": "67105: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F-0/<O:1000-D:463>/=><2-262146-0-dn:41010>"}
cn-0  | 2023/01/11 11:04:32.837245 +0000 DEBUG cn-service morpc/backend.go:591 read response {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "remote": "dn:41010", "backend-id": "3172dc52-56ac-4074-afcc-175c45e444d7", "request-id": 67105, "response": "67105: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F:0/D:834"}
cn-0  | 2023/01/11 11:04:32.837295 +0000 DEBUG cn-service util/log.go:143 txn received responses {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "responses": "[67105: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F:0/D:834]"}
cn-0  | 2023/01/11 11:04:32.837503 +0000 DEBUG cn-service util/log.go:33 txn read {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "txn": "57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0"}
cn-0  | 2023/01/11 11:04:32.837542 +0000 DEBUG cn-service util/log.go:118 txn send requests {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "requests": "[0: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F-0/<O:1000-D:463>/=><2-262146-0-dn:41010>]"}
cn-0  | 2023/01/11 11:04:32.837571 +0000 DEBUG cn-service morpc/backend.go:440 write request {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "remote": "dn:41010", "backend-id": "3172dc52-56ac-4074-afcc-175c45e444d7", "request-id": 67106, "request": "67106: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F-0/<O:1000-D:463>/=><2-262146-0-dn:41010>"}
cn-0  | 2023/01/11 11:04:32.838114 +0000 DEBUG cn-service morpc/backend.go:591 read response {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "remote": "dn:41010", "backend-id": "3172dc52-56ac-4074-afcc-175c45e444d7", "request-id": 67106, "response": "67106: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F:0/D:834"}
cn-0  | 2023/01/11 11:04:32.838162 +0000 DEBUG cn-service util/log.go:143 txn received responses {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "responses": "[67106: <57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0>/Read/F:0/D:834]"}
cn-0  | 2023/01/11 11:04:32.838358 +0000 INFO frontend/util.go:515 time of Exec.Build : 8.122102ms connectionId 1006
cn-0  | 2023/01/11 11:04:32.838750 +0000 INFO frontend/util.go:515 rowCount 2 
cn-0  | time of getDataFromPipeline : 7µs 
cn-0  | processBatchTime 5.6µs 
cn-0  | row2colTime 0s 
cn-0  | allocateOutBufferTime 300ns 
cn-0  | outputQueue.flushTime 0s 
cn-0  | processBatchTime - row2colTime - allocateOutbufferTime - flushTime 5.3µs 
cn-0  | restTime(=tTime - row2colTime - allocateOutBufferTime) 6.7µs 
cn-0  | protoStats flushCount 0 writeCount 1 
cn-0  | writeBytes 0 0 MB
cn-0  |  connectionId 1006
cn-0  | 2023/01/11 11:04:32.839001 +0000 INFO frontend/util.go:515 time of Exec.Run : 516µs connectionId 1006
cn-0  | 2023/01/11 11:04:32.839118 +0000 DEBUG frontend/util.go:520 CommitTxn txnId:57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0 connectionId 1006
cn-0  | 2023/01/11 11:04:32.839188 +0000 DEBUG cn-service util/log.go:51 txn commit {"uuid": "dd1dccb8-4d3c-41f9-b482-5251dc7a41bf", "txn": "57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0"}
cn-0  | 2023/01/11 11:04:32.839212 +0000 DEBUG frontend/util.go:520 CommitTxn exit txnId:57db5547754c43c1b16b70f8a1e3dbcc/Active/S:1673435072830298281-0 connectionId 1006
cn-0  | 2023/01/11 11:04:32.839269 +0000 INFO frontend/util.go:500 query trace status {"connection_id": 1006, "statement": "select * from text_01 where t2 > (select t2 from text_05 where t1 = \"789\")", "status": "success", "span": {"trace_id": "ba364e7f-919f-11ed-ae6c-0242b3130005", "kind": "statement"}, "session_info": "connectionId 1006"}

```

**Comment by daviszhen** [comment]
in reproducing.

**Comment by daviszhen** [comment]
reproduced:

the log denotes the batch.Zs is wrong. 

so,  [yef&&190  wwww  983] is outputed three times.

but the rowCount evaluted from vector.length(bat.vec[0]) is 2.



All log :https://github.com/matrixorigin/matrixone/actions/runs/3912811370

```
cn-0  | 2023/01/13 15:59:07.071479 +0000 INFO frontend/util.go:515 time of Exec.Build : 19.556646ms connectionId 1010
cn-0  | 2023/01/13 15:59:07.072534 +0000 INFO frontend/mysql_cmd_executor.go:912 batch Zs : [3 1]
cn-0  | 2023/01/13 15:59:07.072653 +0000 INFO frontend/mysql_cmd_executor.go:1006 duplicate row
cn-0  | 2023/01/13 15:59:07.072700 +0000 INFO frontend/mysql_cmd_executor.go:640 raw result row 0 : [yef&&190  wwww  983]
cn-0  | 2023/01/13 15:59:07.072784 +0000 INFO frontend/mysql_cmd_executor.go:651 encoded result row 0 : 
cn-0  | 2023/01/13 15:59:07.072889 +0000 INFO frontend/mysql_cmd_executor.go:624 [14 00 00 00 08 79 65 66 26 26 31 39 30 06 20 77 77 77 77 20 03 39 38 33 ]
cn-0  | 2023/01/13 15:59:07.072972 +0000 INFO frontend/mysql_cmd_executor.go:1006 duplicate row
cn-0  | 2023/01/13 15:59:07.073013 +0000 INFO frontend/mysql_cmd_executor.go:640 raw result row 1 : [yef&&190  wwww  983]
cn-0  | 2023/01/13 15:59:07.073071 +0000 INFO frontend/mysql_cmd_executor.go:651 encoded result row 1 : 
cn-0  | 2023/01/13 15:59:07.073165 +0000 INFO frontend/mysql_cmd_executor.go:624 [14 00 00 01 08 79 65 66 26 26 31 39 30 06 20 77 77 77 77 20 03 39 38 33 ]
cn-0  | 2023/01/13 15:59:07.073256 +0000 INFO frontend/mysql_cmd_executor.go:640 raw result row 2 : [yef&&190  wwww  983]
cn-0  | 2023/01/13 15:59:07.073290 +0000 INFO frontend/mysql_cmd_executor.go:651 encoded result row 2 : 
cn-0  | 2023/01/13 15:59:07.073335 +0000 INFO frontend/mysql_cmd_executor.go:624 [14 00 00 02 08 79 65 66 26 26 31 39 30 06 20 77 77 77 77 20 03 39 38 33 ]
cn-0  | 2023/01/13 15:59:07.073371 +0000 INFO frontend/mysql_cmd_executor.go:640 raw result row 3 : [123 7834 commmmmmment]
cn-0  | 2023/01/13 15:59:07.073403 +0000 INFO frontend/mysql_cmd_executor.go:651 encoded result row 3 : 
cn-0  | 2023/01/13 15:59:07.073433 +0000 INFO frontend/mysql_cmd_executor.go:624 [16 00 00 03 03 31 32 33 04 37 38 33 34 0c 63 6f 6d 6d 6d 6d 6d 6d 6d 65 6e 74 ]
cn-0  | 2023/01/13 15:59:07.073499 +0000 INFO frontend/util.go:515 rowCount 2 
cn-0  | time of getDataFromPipeline : 982.608µs 
cn-0  | processBatchTime 837.807µs 
cn-0  | row2colTime 0s 
cn-0  | allocateOutBufferTime 200ns 
cn-0  | outputQueue.flushTime 0s 
cn-0  | processBatchTime - row2colTime - allocateOutbufferTime - flushTime 837.607µs 
cn-0  | restTime(=tTime - row2colTime - allocateOutBufferTime) 982.408µs 
cn-0  | protoStats flushCount 0 writeCount 1 
cn-0  | writeBytes 0 0 MB
cn-0  |  connectionId 1010
```

**Comment by nnsgmsone** [comment]
fixed by #7645

**Comment by aressu1985** [comment]
fixed
